Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,16 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+        [TestMethod]
+        public void CreateLogger_ClassNameIsNull_RetrurnsNull()
+        {
+            // Arrange
+            LogFactory logFactory = new LogFactory();
+
+            // Act
+            logFactory.CreateLogger(null!);
+
+            // Assert
+        }
```

Comment:
You should add a new test in here to make sure that passing in a file path does not return null. In this test you should use the nameof operator. Here is an example of what me and my partner did: 
```suggestion
        }
        [TestMethod]
    [DataRow(nameof(LogFactoryTests))]
    public void CreateLogger_Configured_ReturnsNotNull(string className)
    {
        // Arrange
        LogFactory factory = new();
        factory.ConfigureFileLogger(Path.GetTempFileName());

        // Act
        BaseLogger logger = factory.CreateLogger(className);

        // Assert
        Assert.IsNotNull(logger);
    }
        
        
        
```